src/components/NewPosition/NewPosition.tsx

```diff
@@ -421,6 +428,28 @@ export const NewPosition: React.FC<INewPosition> = ({
     }
   }
 
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      if (refresherTime > 0 && poolKey !== '') {
+        setRefresherTime(refresherTime - 1)
+      } else {
+        reloadHandler()
```

Comment:
this reloadHandler refreshes only currentPlotTicks, but there is more things to  update. You can base on My previous pr which wasn't merged yet, about refresh buttons. 
https://github.com/invariant-labs/webapp/pull/641/files#diff-3afba2da2a71a65fb1e081b59bd3c007d235766c1850702f1fa4a133a97234bb

Reply:
and Its good to show loaders where data are updated, like balances for example